blosc/bitshuffle-altivec.c

```diff
@@ -592,4 +592,23 @@ int64_t bshuf_untrans_bit_elem_altivec(const void* in, void* out, const size_t s
   return count;
 }
 
-#endif /* defined(__ALTIVEC__) */
+
+const bool is_bshuf_altivec = true;
```

Comment:
I used naming consistent with the functions.
The naming (like the SIMD suffix) could be made consistent between bitshuffle and shuffle functions